frontend/test/specs/requestProcessingControllerSpec.js

```diff
@@ -293,83 +293,4 @@
             expect(requestCtrl.warnPaternityExistence).toBeFalsy();
```

Comment:
remove it and every test that checks warnPaternityExistence once you've removed this flag...